### ISSUE TITLE
chrome.docker ECONNREFUSED

### ISSUE THREAD

**Issue by ivanhuay** [oblador/loki#348]
I'm getting this error running `yarn loki update` on my local M1.

```
FAIL  chrome.docker
       Fetching stories
       connect ECONNREFUSED 127.0.0.1:57061
Command failed with exit code 1: docker kill f324fec81b623330f37643935be8b372b745ba8a92cf19b09e468520124ef3d8
Error response from daemon: Cannot kill container: f324fec81b623330f37643935be8b372b745ba8a92cf19b09e468520124ef3d8: No such container: f324fec81b623330f37643935be8b372b745ba8a92cf19b09e468520124ef3d8
```

I tested using chrome.app and it works but no with docker.

**Comment by glebez** [comment]
Loki currently does not seem to work in Docker on M1 Macs. You can find more details in another issue: https://github.com/oblador/loki/issues/344

**Comment by rwieruch** [comment]
+1. Subscribing to this issue, because the other one got closed.

**Comment by alejandrofdiaz** [comment]
I was able to run loki in docker on M1 using `node:16-alpine3.14` as base image, installing chromium `RUN apk add --no-cache chromium` and [building this image](https://www.docker.com/blog/multi-arch-images/) with `arm64` platform support.

**Comment by marduzca** [comment]
The solution on the other issue doesn't work for me.
Following this one as well hoping to have a solution soon

**Comment by cheyer** [comment]
Just created a PR using the [newly introduced `arm64` version](https://github.com/yukinying/chrome-headless-browser-docker/issues/46) of the [chrome headless browser image](https://hub.docker.com/r/yukinying/chrome-headless-browser-stable/tags/). Have tested it on my MacBook Pro 2020 with M1 chip and it has worked so far.

**Comment by techeverri** [comment]
> Loki currently does not seem to work in Docker on M1 Macs. You can find more details in another issue: #344

Loki itself doesn't require any update to work on laptops with M1. In the other hand, the default Docker image used to run the tests against a container didn't support M1 laptops. But it does now https://github.com/yukinying/chrome-headless-browser-docker/issues/46#issuecomment-1099787883

**Comment by cheyer** [comment]
> But it does now https://github.com/yukinying/chrome-headless-browser-docker/issues/46#issuecomment-1099787883

@techeverri But we need to change the docker image used in Loki from [`yukinying/chrome-headless-browser`](https://hub.docker.com/r/yukinying/chrome-headless-browser/tags) to [`yukinying/chrome-headless-browser-stable`](https://hub.docker.com/r/yukinying/chrome-headless-browser-stable/tags) as only the last one also includes the `arm64` arch.
I think it's frustrating for someone just trying to use Loki on a M1 machine and not knowing why it does not work. What do you think?

**Comment by techeverri** [comment]
> I think it's frustrating for someone just trying to use Loki on a M1 machine and not knowing why it does not work. What do you think?

I agree 👍 

Let's wait for @oblador to accept https://github.com/oblador/loki/pull/382 and release a new version.

**Comment by oblador** [comment]
Released in [0.30.0](https://github.com/oblador/loki/releases/tag/v0.30.0)
